### PR TITLE
Fix missing libraries bug in make bundle

### PR DESCRIPTION
**Pull request by Czaki**
# Description

## Type of change

- [x] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# References

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .github/workflows/make_bundle.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ on:
     # Sequence of patterns matched against refs/tags
     tags:
       - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+  pull_request:
+    branches:
+      - '**'
```
Comment (Czaki) [inline .github/workflows/make_bundle.yml:8]: to allow test if PR works

File: .github/workflows/make_bundle.yml [hunk 2]
```diff
@@ -61,6 +64,7 @@ jobs:
         id: get_release
         uses: bruceadams/get-release@v1.2.0
       - name: Upload Release Asset
+        if: startsWith(github.ref, 'refs/tags/v') != true
```
Comment (Czaki) [inline .github/workflows/make_bundle.yml]: to avoid upload try during PR.

**Review by Czaki — COMMENTED** [review]
Because bundling operation not tested on every commit then I think that there should be some branch that runs these instructions to test it without tagging.

**Comment by Czaki** [comment]
@sofroniewn @ziyangczi fixed

**Comment by tlambert03** [comment]
Thanks!  Do you happen to know why these libs are now necessary when it used to build fine without them?

**Comment by Czaki** [comment]
> Thanks! Do you happen to know why these libs are now necessary when it used to build fine without them?

Since Qt 5.15 part of `xcb` dependencies are not bundled and need to be installed in the system.  See https://codereview.qt-project.org/c/qt/qtbase/+/253905.

**Comment by tlambert03** [comment]
Good to know, thanks again

**Review by sofroniewn — APPROVED** [review]
This looks good to me. I'm fine building every PR, it's better to find out when something is broken right away. It looks like the artifacts don't get uploaded anywhere which is fine.

I'd been discussing with @ziyangczi something like a "nightly build" which would upload the latest bundles to our website, but that can come later.

**Pull request merged**